### ISSUE TITLE
Swipe gestures are too sensitive 

### ISSUE THREAD

**Issue by pentmo** [dessalines/jerboa#1399]
### Jerboa Version

0.0.59

### Android Version + Phone

Google Pixel 4a

### Describe The Bug

When scrolling through posts, I am unintentionally bringing up the left swipe menu on posts rather than scrolling down to the next post as intended. I enabled `only right swipe` in Settings > Look and Feel > `Preset for swipe gesture`. 

Now when I scroll the same way, instead of executing the left swipe gesture, the app will get stuck on posts rather than scrolling down as intended.

### To Reproduce

1. Go to Home
2. Scroll down at a left or right angle

**Comment by dessalines** [comment]
I agree its a little too sensitive, we should probably tweak it a bit.
